sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -252,6 +252,34 @@ public ConsumerFactoryFn(List<String> topics,
     }
   }
 
+  /**
+   * Creates a consumer with two topics, with 5 partitions each.
+   * numElements are (round-robin) assigned all the 10 partitions.
+   */
+  private static KafkaIO.Read<Integer, Long> mkKafkaReadTransformWithCoders(
+          int numElements,
+          @Nullable SerializableFunction<KV<Integer, Long>, Instant> timestampFn) {
+
+    List<String> topics = ImmutableList.of("topic_a", "topic_b");
+
+    KafkaIO.Read<Integer, Long> reader = KafkaIO
+            .<Integer, Long>readWithCoders(VarIntCoder.of(), VarLongCoder.of())
+            .withBootstrapServers("myServer1:9092,myServer2:9092")
+            .withTopics(topics)
+            .withConsumerFactoryFn(new ConsumerFactoryFn(
+                    topics, 10, numElements, OffsetResetStrategy.EARLIEST)) // 20 partitions
```

Comment:
Actually, I have reused the  comment`mkKafkaReadTransform`. The `//20 partitions` comment seems correct since this is 10 partitions per topic and 2 topics, but the method's Javadoc seems to be incorrect. I'll fix that.